tests/MongoDB.Driver.TestHelpers/DriverTestConfiguration.cs

```diff
@@ -67,6 +70,30 @@ public static DatabaseNamespace DatabaseNamespace
         }
 
         // public static methods
+        public static DisposableMongoClient CreateDisposableClient()
```

Comment:
Centralize the creation of DisposableMongoClient in this class. A few other test classes have minor changes to call  these new methods.